Resources/translations/content_type.en.xlf

```diff
@@ -145,6 +145,70 @@
         <source>content_type.is_in_use</source>
         <target>The content type cannot be deleted because it has content items, or because you don't have permission.</target>
       </trans-unit>
+      <trans-unit id="c06173c2f83c7d3b1db6b93751eac959" resname="content_type.metadata.title">
+        <source>content_type.metadata.title</source>
+        <target>Global properties</target>
```

Comment:
We should use the "desc" function from the JMS translation bundle to set the english strings, and use the extractor to generate the xlf file. Keep 'em like this for now, but this part interacts with the work on the crowdin integration.